tests/flot.scattergraph.Test.js

```diff
@@ -129,6 +129,24 @@ describe('A scatter graph', function () {
         expect(ctx.moveTo).toHaveBeenCalled();
         expect(ctx.arc).toHaveBeenCalled();
     });
+    it('should draw points for multiple series', function () {
```

Comment:
This doesn't really seem like the best test, but I haven't come up with something better yet.